dnf/cli/commands/module.py

```diff
@@ -271,6 +271,28 @@ def run_on_module(self):
 
             logger.error(dnf.exceptions.MarkingErrors(no_match_group_specs=skipped_groups))
 
+    class SwitchToSubCommand(SubCommand):
+
+        aliases = ('switch-to',)
+        summary = _('switch module to streams and distrosync rpm packages')
```

Comment:
This description no longer passes the grammar check :slightly_smiling_face: what about:
> switch a module to a different stream and distrosync rpm packages

Reply:
What about `switch a module to a stream and distrosync rpm packages`